hooks/pre-commit

```diff
@@ -21,6 +41,7 @@ unformatted=$(gofmt -l $gofiles)
 # run gofmt on improperly formatted files.
 for fn in $unformatted; do
 	gofmt -w -e $PWD/$fn
```

Comment:
`-e` flag isn't really necessary here since we just want to reformat the files directly. Output is not *really* necessary.